kaspresso/src/main/kotlin/com/kaspersky/kaspresso/interceptors/behavior/impl/autoscroll/FallbackAutoScrollToAction.kt

```diff
@@ -0,0 +1,91 @@
+package com.kaspersky.kaspresso.interceptors.behavior.impl.autoscroll
+
+import android.util.Log
+import android.view.View
+import android.view.ViewParent
+import android.widget.HorizontalScrollView
+import android.widget.ListView
+import android.widget.ScrollView
+import androidx.core.widget.NestedScrollView
+import androidx.test.espresso.PerformException
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.util.HumanReadables
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+
+class FallbackAutoScrollToAction : ViewAction {
```

Comment:
@Vacxe Can you check this solution? I am not a great specialist in Espresso tricks =)

Reply:
@Vacxe ping =)